pkg/preparer/orchestrate.go

```diff
@@ -68,7 +68,7 @@ func WatchForPodManifestsForNode(nodeName string, consulAddress string, hooksDir
 
 // no return value, no output channels. This should do everything it needs to do
 // without outside intervention (other than being signalled to quit)
-func handlePods(hooks *pods.HookDir, podChan <-chan pods.PodManifest, quit <-chan struct{}, logger logging.Logger) {
+func handlePods(hooks *pods.HookDir, podChan <-chan pods.PodManifest, quit <-chan struct{}, logger logging.Logger, store *intent.Store) {
```

Comment:
this arg list is getting out of hand pretty fast. next opportunity lets change this to a function on a struct with these things instead.